string                          name
geometry_msgs/TransformStamped  position
float64[]                       boundingbox_2d
